Snowflake/Snowflake_Basics_and_data_querying.ipynb

```diff
@@ -0,0 +1,732 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snowflake - Basic usage and data querying\n",
+    "\n",
+    "<a href=\"https://app.naas.ai/user-redirect/naas/downloader?url=https://raw.githubusercontent.com/jupyter-naas/awesome-notebooks/master/Snowflake/Snowflake_Basics_and_data_querying.ipynb\" target=\"_parent\"><img src=\"https://naasai-public.s3.eu-west-3.amazonaws.com/open_in_naas.svg\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Tags:** #snowflake #data #warehouse #naas_drivers #snippet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Author:** [Mateusz Polakowski](https://www.linkedin.com/in/polakowski/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook shows basic usage of Snowflake driver.\n",
+    "\n",
+    "Below you can find essential operations for setting up the environment and querying data that already exists in your data warehouse."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from naas_drivers import snowflake\n",
+    "from snowflake.connector.errors import ProgrammingError"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setup Snowflake account\n",
+    "\n",
+    "If you don't have your SF account, you can easily set up a [30-day trial account with $400 budget here](https://signup.snowflake.com/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Credentials"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Here environment variables are used to pass Snowflake credentials, \n",
+    "# but it's okay to do it in a different manner\n",
+    "\n",
+    "sf_username=os.environ['SNOWFLAKE_USER']\n",
+    "sf_password=os.environ['SNOWFLAKE_PASSWORD']\n",
+    "sf_account=os.environ['SNOWFLAKE_ACCOUNT']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connecting to your Snowflake account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "snowflake.connect(\n",
```

Comment:
I am getting a timeout everytime. I followed the instructions from the driver. It could be nice to tell here how to get the `sf_account` value as it's not documented.

Reply:
<img width="1359" alt="image" src="https://user-images.githubusercontent.com/22365519/185784574-4e174a5d-bc8d-4107-af09-c49c41b8c63a.png">

Reply:
My account is `qd55120`

Reply:
@Dr0p42 you have to also specify the region where your account is located. For my trial account it was for example `bq66451.eu-central-1`. 

There are also several other methods to pass this parameter. The one we're discussing can be found in [snowflake accounts documentation](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#using-an-account-locator-as-an-identifier), next to other account values that can be passed.

Reply:
It worked thank you :)